### PR TITLE
89908a Removed unused marriage schema logic - form 10-10d

### PR DESCRIPTION
**Pull request by michaelclement**
## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

Removed the schemas, tests, and `depends` functions related to the additional marriage/separation date information in form 10-10d.

## Related issue(s)

- NA

## Testing done

- Manual
- Unit

## Screenshots

NA

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

NA

### CODE REVIEW

**Review by reiting — DISMISSED** [review]

**Review by balexandr — DISMISSED** [review]

**Review by Andrew565 — DISMISSED** [review]
Approved, but you should consider deleting commented-out code. It'll still be there in the Git history if you need to get it back.

**Review by cloudmagic80 — DISMISSED** [review]

File: src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx:1]: In the previous commit, the code deleted here was simply commented out. Now it's being fully removed.

**Review by balexandr — APPROVED** [review]

**Comment by michaelclement** [comment]
@Andrew565 - updated per your comment to delete rather than just comment. Also went ahead and pulled some related stuff that had been previously commented out. Could I please get a re-review? Thanks!

**Review by cloudmagic80 — APPROVED** [review]

**Review by Andrew565 — APPROVED** [review]
Approved but you should consider removing comments, easy enough to bring that code back from the git history if needed.

**Pull request merged**